Mathlib/Data/Finset/LocallyFinite.lean

```diff
@@ -67,6 +67,7 @@ theorem nonempty_Ioc : (Ioc a b).Nonempty ↔ a < b := by
   rw [← coe_nonempty, coe_Ioc, Set.nonempty_Ioc]
 #align finset.nonempty_Ioc Finset.nonempty_Ioc
 
+-- TODO: This is nonsense. A locally finite order is never densely ordered
```

Comment:
Can you ping the author of this lemma (and link the PR that added it) and check that they agree before merging?

Reply:
I was the author in https://github.com/leanprover-community/mathlib/pull/9495 😅

Reply:
Do we have the statement that this is nonsense? Perhaps in the form that intervals in a dense order are infinite